TwoPageEBookReader/app/src/main/res/layout/book_page_layout_split_horizontal.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~  Licensed under the MIT License.
+  -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="horizontal"
+    android:background="?android:colorBackground">
+
+    <LinearLayout
+        android:layout_width="0dp"
+        android:layout_height="match_parent"
+        android:layout_weight="0"
+        android:orientation="vertical"
+        android:tag="page_container_0">
+
+        <LinearLayout
+            android:id="@+id/linear_layout"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_weight="1"
+            android:orientation="vertical"
+
+            android:padding="@dimen/page_padding" />
+
+        <include layout="@layout/book_page_text_caption"
+            android:id="@+id/caption_view"/>
+
+    </LinearLayout>
+
+    <View
+        android:id="@+id/page_divider"
+        android:layout_width="0dp"
+        android:layout_weight="1"
+        android:layout_height="match_parent"/>
+
+    <LinearLayout
+        android:layout_width="0dp"
+        android:layout_height="match_parent"
+        android:layout_weight="0"
+        android:orientation="vertical"
+        android:tag="page_container_1">
+
+        <LinearLayout
+            android:id="@+id/linear_layout2"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_weight="1"
+            android:orientation="vertical"
+
+            android:padding="@dimen/page_padding" />
+
+        <include layout="@layout/book_page_text_caption"
+            android:id="@+id/caption_view2"/>
+
+    </LinearLayout>
+
+</LinearLayout>
```

Comment:
probably you can use a `ConstraintLayout` and use constraints to align your views instead of using multiple `LinearLayouts`. Multiple layouts will make that the platform works harder to render your layouts and eventually you can get dropped frames (probably not here, but as learning for more complex layout you build in the future :) )

Reply:
good to know. In this situation, flattening the layouts would take some work, as part of the code relies on setting the size of these layouts.

Reply:
Sure, just keep it in mind :-)